drizzlepac/runsinglehap.py

```diff
@@ -53,13 +53,27 @@ def perform(input_filename, **kwargs):
     debug : Boolean
         display all tracebacks, and debug information?
 
+    log_level : string
+        The desired level of verboseness in the log statements displayed on the screen and written to the .log file.
+
     Updates
     -------
     return_value : list
         a simple status value. '0' for a successful run and '1' for a failed
         run
     """
-    log.info("Starting single-visit processing of {}".format(input_filename))
+    # set up log_level as an input to hapsequencer.run_hap_processing().
+    if kwargs['log_level'] == "critical":
+        kwargs['log_level'] = logutil.logging.CRITICAL
+    elif kwargs['log_level'] == "error":
+        kwargs['log_level'] = logutil.logging.ERROR
+    elif kwargs['log_level'] == "warning":
+        kwargs['log_level'] = logutil.logging.WARNING
+    elif kwargs['log_level'] == "INFO":
+        kwargs['log_level'] = logutil.logging.INFO
+    else:
+        kwargs['log_level'] = logutil.logging.DEBUG
+
```

Comment:
Do we really want the default logging mode to be DEBUG?  I would have thought the default should be INFO.

Reply:
issue resolved by commit [889def1](https://github.com/spacetelescope/drizzlepac/pull/506/commits/889def19be2f0eb53bd402492bd61354d0a03474)